client/doc.go

```diff
@@ -0,0 +1,85 @@
+/*
+Package client provides NeoFS API client implementation.
+
+The main component is Client type. It is a virtual connection to the network
+and provides methods for executing operations on the server.
+
+Create client instance:
+	var c client.Client
+
+Initialize client state:
+	var prm client.PrmInit
+	prm.SetDefaultPrivateKey(key)
+	// ...
+
+	c.Init(prm)
+
+Connect to the NeoFS server:
+	var prm client.PrmDial
+	prm.SetServerURI("localhost:8080")
+	prm.SetDefaultPrivateKey(key)
+	// ...
+
+	err := c.Dial(prm)
```

Comment:
It is unclear what the difference between `Init()` and `Dial()`. I suggest to drop `Init()` and keep `New()` constructor with parameters there.

Reply:
`Init` initializes struct, `Dial` opens the connection. Methods are documented.

Reply:
Proposed `New` function doesn't make sense IMO. What can be useful is a function that allows you to quickly build a `Client` and connect to the server, for example `New(address string) (*Client, error)`.